app/controllers/home_controller.rb

```diff
@@ -1,5 +1,7 @@
 class HomeController < ApplicationController
   def index
     @contact = Contact.new
+    @developers = Admin::Developer.where(active: true)
```

Comment:
Is better to use [scope](http://guides.rubyonrails.org/active_record_querying.html#scopes), it let your test more easy.

```
class Admin::Developer < ApplicationRecord
  scope :actives, -> { where(active: true) }
end
```
In the controller
```
@developers = Admin::Developer.actives
```